src/style.css

```
@@ -1 +1,73 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+}
+
+.nav__list,
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  padding: 0;
+  flex-grow: 1;
+}
+
+.nav__item {
+  display: flex;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-family: Roboto, serif;
```

```suggestion
  font-family: "Roboto", sans-serif;
```
roboto - це шрифт без засічок